README.md

```diff
@@ -393,6 +394,32 @@ Used when you want to build the app to production programically. It is adviced t
 ViteExpress.build();
 ```
 
+### `async getViteConfig() => Promise<ViteConfig>`
+
+Useful in case you want to write a specific `console.log` message in the listen callback, for example to construct a URL using the base too.
```

Comment:
I would change that to:
> Useful in case you want to know the Vite configuration parameters currently in use, for example when you want to write a specific `console.log` message with that information in the listen callback.

Something like that, otherwise it looks good!